pkg/envoy/ads/stream_test.go

```diff
@@ -82,3 +84,43 @@ func TestMapsetToSliceConvFunctions(t *testing.T) {
 	assert.True(findSliceElem(nameSlice, "C"))
 	assert.False(findSliceElem(nameSlice, "D"))
 }
+
+var _ = Describe("Test ADS response functions", func() {
```

Comment:
Would you mind using Go testing instead of Ginkgo as we've been trying to move away from Ginkgo for unit tests.